_posts/2020-12-24-how-to-start-with-foreman

```diff
@@ -0,0 +1,399 @@
+
+Unmanaged Hosts can be used in many use cases. The first and most simple use case is the inventory.
+You can keep track of your infrastructure information. You can have multiple [users and control](https://theforeman.org/manuals/latest/index.html#4.1.2RolesandPermissions),
+who has access to specific subset of data. This data can be searched through
+using advanced, yet easy to use, search syntax. You can generate various reports
+and statistics. Therefore you can e.g. easily tell, how many RHEL 7 and 8 are
+running in your datacenter, search for all hosts in *Subnet* 192.168.0.1/24,
+create CSV list of all hosts that are owned by a specific group of admins.
+
+Once you have hosts defined, you can also take advantage of some nice plugins,
+such as [remote execution](https://www.theforeman.org/plugins/foreman_remote_execution/1.7/index.html)
+to run scripts and commands via SSH. Or run [ansible](https://www.theforeman.org/plugins/foreman_ansible/2.x/index.html)
+playbooks. You can setup your configuration management tool, to send more
+information about your machines to Foreman and use Foreman as a dashboard for your
+infrastructure status. You can receive email notifications in case some configuration
+run fails. You can also manage the configuration management policy, be it Ansible, Puppet,
+Salt or Chef. With [Katello](https://theforeman.org/plugins/katello/3.18/index.html) plugin,
+you can watch what packages are upgradable or what security vulnerabilities were detected and
+can be patched on your hosts. You can even manage what package versions or files your hosts
+have access to. You can define OpenSCAP based compliance policy and start enforcing it across
+your infrastructure.
+
+All these use-cases deserve it's own post, but we'll start first with how to
+begin, or better to say, how to create some hosts.
+
+### Adding unmanaged hosts manually
+
+The easiest way to create an unmanaged host is using [Hammer CLI](https://www.theforeman.org/manuals/latest/index.html#4.5CommandLineInterface).
+Once Hammer is installed and configured, you can create a host with this command
+
+```
+hammer host create --name my-first-host \
+  --managed false \
+  --build false \
+  --organization 'Default Organization' \
+  --location 'Default Location'
+```
+
+Name is usually the FQDN of the host, but it can be also just a hostname without domain.
+The `managed` parameter tells Foreman, this will be unmanaged host. The build value
+represents whether the host is currently in build mode, which is only possible
+for managed hosts. Without explaining what exactly it means, it always needs to be set
+to false for unmanaged hosts, otherwise the host record is considered invalid.
+
+More parameters, such as `--ip`, `--owner`, `--comment`, `--model` can be specified.
+For the full list, run `hammer host create --help`
+
+The other option is to use our API directly. In fact, all hammer commands are using
```

Comment:
and you do not mention FAM? blasphemy!

Reply:
the second post has the collection mentioned, but it's not the most straightforward to describe, so I would prefer to avoid it, I think everyone already read about this in the last 10 blog posts :-)